synapse/rest/client/v2_alpha/register.py

```diff
@@ -188,6 +213,81 @@ def on_POST(self, request):
         )
 
 
+class RegistrationSubmitTokenServlet(RestServlet):
+    """Handles registration 3PID validation token submission"""
+
+    PATTERNS = client_patterns(
+        "/registration/(?P<medium>[^/]*)/submit_token/*$", releases=(), unstable=True
```

Comment:
no need to accept trailing slashes
```suggestion
        "/registration/(?P<medium>[^/]*)/submit_token$", releases=(), unstable=True
```